charts/nri-prometheus/tests/deployment_test.yaml

```diff
@@ -41,3 +41,14 @@ tests:
           value: 
             foo: bar
   
+  - it: adds securityContext if specified in values.
```

Comment:
Are we falling again into retesting everything instead of adding a simple installability test to a `ci/*.yaml`?

Reply:
I will have a look at what is being tested in with ct and remove the unit tests which doesn't make sense to actually test what is being rendered. Having in mind that with the unit test we can assert the values that are rendered while in the ct tests we just tests that is installable i think.

Reply:
Done!